lib/services/packagist.rb

```diff
@@ -37,14 +36,13 @@ def domain
 
   def full_domain
     if data['domain'].to_s == ''
-      'http://packagist.org'
+      'https://packagist.org'
     else
       data['domain']
-    end.lstrip.sub(/[\/\s]+$/,'')
+    end.lstrip.sub(/[\/\s]+$/,'').sub(/^http:\/\/packagist.org/, 'https://packagist.org')
```

Comment:
@Seldaek with the `^` and `$`, it'd be possible to add a newline into the string and still get a correct match since those match per "line" and not "per string".